app/models/ticket.js

```diff
@@ -44,5 +44,31 @@ export default ModelBase.extend({
 
   itemTotal: computed('price', 'quantity', function() {
     return this.price * this.quantity;
+  }),
+
+  /**
+   * This attribute computes total ticket price payable after inclusion
+   * of additional taxes on the base ticket price
+   */
+  ticketPriceWithTax: computed('event', function() {
+    let taxType = this.event.get('tax.isTaxIncludedInPrice');
+    if (!taxType) {
+      return ((1 + this.event.get('tax.rate') / 100) * this.price).toFixed(2);
+    }
+    return this.price;
+  }),
+
+  /**
+   * This attribute computes value added tax amount in the cases
+   * when tax amount is included in ticket price, otherwise return
+   * 0
+   */
+  includedTaxAmount: computed('event', function() {
```

Comment:
Please be specific in computed properties. Watch for the exact property you are using. In this case, `event.tax.isTaxIncludedInPrice` and `event.tax.rate`

Reply:
apologies for repeating this mistake. Will take care from next time.